### PR TITLE
Simplify download instructions for Roslyn README

### PR DESCRIPTION
**Pull request by agocke**
@dotnet/roslyn Trying to clean up the README, please take a look

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,28 +22,19 @@ The .NET Compiler Platform ("Roslyn") provides open-source C# and Visual Basic c
 with rich code analysis APIs.  It enables building code analysis tools with the same APIs
```
Comment (jaredpar) [inline README.md]: Can the line above just say Roslyn?

File: README.md [hunk 2]
```diff
@@ -22,28 +22,19 @@ The .NET Compiler Platform ("Roslyn") provides open-source C# and Visual Basic c
...
-
-Get started building diagnostics, code fixes, refactorings, and other code-aware tools!
-
-- [**Getting Started On Visual Studio 2015**](https://github.com/dotnet/roslyn/wiki/Getting-Started-on-Visual-Studio-2015)
-
-Or, you can grab the latest [NuGet Roslyn compiler package](http://www.nuget.org/packages/Microsoft.CodeAnalysis). 
-From the NuGet package manager console:
-
-    Install-Package Microsoft.CodeAnalysis
+To get the latest "preview" drop, published about once per month, add the `-pre` switch to the nuget commands.
```
Comment (amcasey) [inline README.md]: Does it matter where "-pre" goes?  I'd be tempted to include an example.

Comment (jaredpar) [inline README.md]: I would say "once per sprint"

**Comment by jaredpar** [comment]
:+1:

File: README.md (hunk 2 above)
Comment (ManishJayaswal) [inline README.md]: and mention that our sprints are three weeks long.

File: README.md (hunk 1 above)
Comment (agocke) [inline README.md]: I would like it to. If it can is up to you :)

File: README.md [hunk 3]
```diff
@@ -22,28 +22,19 @@ The .NET Compiler Platform ("Roslyn") provides open-source C# and Visual Basic c
 with rich code analysis APIs.  It enables building code analysis tools with the same APIs 
 that are used by Visual Studio.
 
-### Try new language and IDE features
+### Download C# and Visual Basic
 
-Just want to provide feedback on new [language features](https://github.com/dotnet/roslyn/wiki/Languages-features-in-C%23-6-and-VB-14) 
-and [IDE features](http://blogs.msdn.com/b/visualstudio/archive/2014/11/12/the-c-and-visual-basic-code-focused-ide-experience.aspx)? 
+Want to start developing in C# and Visual Basic? Download [Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/visual-studio-2015-downloads-vs.aspx), 
+which has the latest features built-in. There are also [prebuilt Azure VM images](http://blogs.msdn.com/b/visualstudioalm/archive/2014/06/04/visual-studio-14-ctp-now-available-in-the-virtual-machine-azure-gallery.aspx) available with VS 2015 already installed.
```
Comment (pharring) [inline README.md]: The link refers to pre-release versions. Try this link instead:
https://azure.microsoft.com/en-us/marketplace/virtual-machines/all/?term=Visual+Studio+2015

**Comment by TyOverby** [comment]
lgtm

**Pull request merged**